app/components/mobile-dialog.hbs

```diff
@@ -0,0 +1,8 @@
+{{#if this.toShow}}
+  <dialog class="appDialog">
+      Open RDS in app
+    <br />
+    <button id="cancleBt" type="button" {{on 'click' (fn this.closeDialog)}} >Cancle</button>
```

Comment:
typo in id, also what is bt? 
Doesn't clariy the meaning, please don't use abbreviations.